### PR TITLE
finished task

### PR DESCRIPTION
**Pull request by IvanovIurii**
Added Google formatter to format Page Object files in accordance with Google Style Guide.

### CODE REVIEW

File: src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java [hunk 1]
```diff
@@ -17,12 +23,14 @@
...
 
+    private List<JavaFile> javaFiles = new ArrayList<>();
+
     private final static Logger logger = LoggerFactory.getLogger(JavaFileWriter.class);
 
     /**
      * Calls a method to write each {@link JavaClass} in outputDir folder.
      *
-     * @param outputDir path to the folder where need to generate .java source files.
+     * @param outputDir   path to the folder where need to generate .java source files.
```
Comment (vladgogit) [inline src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java]: no need in extra space

**Review by vladgogit — COMMENTED** [review]
it looks fine to me but need to be approved by some DEV reviewers

File: src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java (hunk 1 above)
Comment (hdghg) [reply src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java]: These spaces were added by code formatter

**Review by hdghg — APPROVED** [review]

File: src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java [hunk 2]
```diff
@@ -42,16 +50,63 @@ public void writeFiles(String outputDir, List<JavaClass> javaClasses) throws IOE
...
-        JavaFile.builder(javaClass.getPackageName(), buildTypeSpec(javaClass))
-            .build()
-            .writeTo(Paths.get(outputDir));
+        JavaFile javaFile = getJavaFileFromClass(javaClass);
+        try {
+            String formattedFile = new Formatter().formatSource(javaFile.toString());
+            Path path = this.getFullPath(Paths.get(outputDir), javaFile);
+            Files.write(path, formattedFile.getBytes());
+        } catch (FormatterException e) {
+            logger.warn("There was an error during format a source: " + e.getMessage());
```
Comment (kugushev) [inline src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java:59]: Do you really need to suppress stack trace?

Comment (IvanovIurii) [reply src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java:59]: no, was made accidently, will fix

**Pull request merged**